contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
@gierschv circleci should be able to handle ceph-deploy. If you need help, the people on the support chat at https://circleci.com/ are very helpful.